CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+cmake_minimum_required (VERSION 3.5)
+
+# Common configuration
```

Comment:
cmake --build . --target RunTests failing for out of src builds when I build from a build sub-folder.  It can't find the test files.